### ISSUE TITLE
Tool help section title font sizes could need improvement

### ISSUE THREAD

**Issue by bernt-matthias** [galaxyproject/galaxy#21679]
**Describe the bug**

The help structured along the lines of (important are 3 levels of sections and a definition list):

```
breaks
=============

...

breaks.flagIsolated
-----------------------------
        
...

Parameters
\`\`\`\`\`\`\`\`\`\`\`\`\`

field
    Variable to process.
gap_window : str
    Minimum gap size required before and after a data group to consider it
    isolated. See conditions (2) and (3) below.
```

renders as:

<img width="300" height="498" alt="Image" src="https://github.com/user-attachments/assets/7d6809d9-f199-45a1-869e-6ad7e5e12862" />

- The font size of the 3rd level section title is already smaller than the font size of the text
- Definition list is more prominent then section titles

**Galaxy Version and/or server at which you observed the bug**
Galaxy Version: 25.1

**Browser and Operating System**
Operating System: Linux
Browser: Firefox

**Expected behavior**

- Section titles should be more prominent than text (maybe underline as well?)
- Definition lists should not look like section titles

**Comment by hujambo-dunia** [comment]
uiux mtg decision: restore standard header format; only show last 3 headers in section; thus, h5=h6.

**Comment by mvdbeek** [comment]
Fixed in https://github.com/galaxyproject/galaxy/pull/21780
